pkg/images/defaults/flavor.go

```diff
@@ -218,8 +218,7 @@ func OpenSourceImageFlavor() ImageFlavor {
 		ScannerDBSlimImageName: "scanner-db-slim",
 
 		ChartRepo: ChartRepo{
-			// TODO(ROX-10079): establish a place where open source users will be able to download Helm charts.
-			URL: "https://charts.stackrox.io",
+			URL: "https://github.com/stackrox/helm-charts/opensource/",
```

Comment:
This needs to change now as we're using ...raw.githubusercontent...
To see why it matters, how and where it is used, you can do the following:
1. Run `make cli`, unless you've already done so.
2. Run `roxctl helm output central-services --image-defaults opensource --output-dir blah`.
3. Examine `blah/README.md` file produced by the last command. You'll find mention of `helm repo add` and this URL there.
 
See commands you used in testing to figure out the proper URL to put.

Reply:
Makes sense, I'll test it but I expect the correct repo link to be `https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/` then

Reply:
Yeah looks good.